# This message is consisted of a stae for path planning
# x,y,z,yaw,drones_distance,drones_angle
float32 x
float32 y
float32 z
float32 yaw
float32 drones_distance
float32 drones_angle